index.html

```diff
@@ -0,0 +1,216 @@
+            <p class="about-info"> I can help you build a product , 
+            feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+            don’t hestiate to contact me.</p>
+            <button class="about-btn" type="button">Get My Resume</button>
+        </div>
+        <div class="about-wrapper">
+            <div class="cir-img"><img src="images/circle.png" alt="A circle that shows languages"></div>
+            <h1 class="about-head">Languages</h1>
+            <ul class="works-details">
+                <li>JavaScript</li>
+                <li>Ruby</li>
+                <li>Html</li>
+                <li>Css</li>
+            </ul>
+        </div>
+        <div class="about-wrapper">
+            <div class="cir-img"><img src="images/circle2.png" alt="A circle that shows frames"></div>
+            <h1 class="about-head">Frameworks</h1>
+            <ul class="works-details">
+                <li>Bootstrap</li>
+                <li>Ruby on rails</li>
+                <li>RSpec</li>
+                <li>Capybara</li>
+                <li>Selenium</li>
+            </ul>
+        </div>
+
+        <div class="about-wrapper">
+            <div class="cir-img"><img src="images/skill.png" alt=""></div>
+            <h1 class="about-head">Skills</h1>
+            <ul class="works-details">
+                <li>Codekit</li>
+                <li>GitHub</li>
+                <li>Codepen</li>
+                <li>Gitlab</li>
+                <li>Terminal</li>
+            </ul>
+        </div>  
+    </section>
+      <!--- Footer -->
+    <footer class="footer-container">
+        <div class="footer-title">
+            <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+        </div>
+        
+        <form action="#" class="footer-form">
+            <input class="input" type="text" placeholder="e.g Sollomon Teka">
+            <input class="input" type="email" placeholder="e.g sollomonteka@gmail.com">
+            <textarea name="" id="" cols="30" rows="10">Enter text here</textarea>
+            <button class="about-btn" type="button">Get in touch</button>
+        </form>  <!-- main end-->
+
+        <!--- End of form-->
+        <div class="media-link">
+            <ul class="socials">
+                <li><a href="#"><img src="images/twitter.png" alt="A twitter image"></a></li>
+                <li><a href="#"><img src="images/Linkedin icon.png" alt="A Linkedin image"></a></li>
+                <li><a href="#"><img src="images/medium.png" alt="A medium image"></a></li>
+                <li><a href="#"><img src="images/github.png" alt="A github image"></a></li>
+                <li><a href="#"><img src="images/angellist.png" alt="A angellist image"></a></li>
+            </ul>
+            <a href="mailto:www.gmail.com">yoriyordina7@gmail.com</a>
+        </div>
+    </footer>
+    
```

Comment:
I appreciate your effort adding this part, but I think for now you should focus more on the required session and styling of this can come when it is time.☺